src/components/ifxAccordion/ifxAccordion.tsx

```diff
@@ -0,0 +1,28 @@
+//ifxAccordion.tsx
+import { Component, h, Listen } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-accordion',
+  styleUrl: 'ifxAccordion.scss',
+  shadow: true,
+})
+export class IfxAccordion {
+  @Listen('itemOpened', { target: 'body' })
+  async onItemOpened(event: CustomEvent) {
+    const items = Array.from(document.querySelectorAll('ifx-accordion-item'));
+    for (const item of items) {
+      const itemElement = item as HTMLIfxAccordionItemElement;
+      if (itemElement !== event.target && (await itemElement.isOpen())) {
+        itemElement.close();
+      }
+    }
```

Comment:
Because you are querying the document, if you have more than 1 accordion in your app, you will grab all of the items from all the accordions. While it is clear you are matching the `event.target` to find the right element, it is not clear why have you decided to do it this way, and not simply query the component itself. 
Eg:  Use  `@Element() el`; and then  `this.el.querySelectorAll()`